Kickstarter-iOS/Features/Messages/Controller/MessagesViewController.swift

```diff
@@ -170,16 +189,26 @@ extension MessagesViewController: BackingCellDelegate {
 // MARK: - MessageCellDelegate
 
 extension MessagesViewController: MessageCellDelegate {
-  func messageCellDidTapHeader(_ cell: MessageCell, _: User) {
+  func messageCellDidTapHeader(_ cell: MessageCell, _ author: User) {
```

Comment:
Nit: I don't think `author` is the right name in this context - just `user` would be better